src/pdsfile/_properties.py

```
@@ -0,0 +1,1686 @@
+    @property
+    def volume_data_set_ids(self):
+        """Return a list of the dataset IDs found in this volume."""
+
+        if self._volume_data_set_ids_filled is None:
+            self._volume_data_set_ids_filled = self._volume_info[4]
+            self._recache()
+
+        return self._volume_data_set_ids_filled
+
+    @property
+    def version_ranks(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `version_ranks`

(ANN202)